test/client/bespoke-client-test.ts

```diff
@@ -37,11 +37,21 @@ describe("BespokeClient", function() {
 
         it("Connects to something other than localhost", function(done) {
             this.timeout(5000);
-            const client = new BespokeClient("JPK" + nodeMajorVersion, "proxy.bespoken.tools", 5000, "0.0.0.0", testPort);
+            const client = new BespokeClient("JPK" + nodeMajorVersion,
+                "proxy.bespoken.tools",
+                5000,
+                "127.0.0.1",
+                testPort);
+
             client.onConnect = function (error: any) {
                 const webhookCaller = new HTTPClient();
-                webhookCaller.post("proxy.bespoken.tools", 443, "/test?node-id=JPK" + nodeMajorVersion, "Test", function (data: Buffer, statusCode: number, success: boolean) {
-                    assert.equal(data.toString(), "BST Proxy - Local Forwarding Error\nconnect ECONNREFUSED 0.0.0.0:" + testPort);
+                webhookCaller.post("proxy.bespoken.tools",
```

Comment:
are those actually blank lines? :smiley:

Reply:
Line was to big, so i added a line jump between attributes